1.4/Languages/Japanese/Keyed/FarmingHysteresis.xml

```diff
@@ -1,33 +1,54 @@
-  <FarmingHysteresis.EnableFarmingHysteresisisDesc>設定された下限値を在庫が下回った場合に種蒔きを開始して,設定された上限値を在庫が超えた場合に種蒔きを停止します.</FarmingHysteresis.EnableFarmingHysteresisisDesc>
-
-  <FarmingHysteresis.UseGlobalValues>同種在庫 全域対応</FarmingHysteresis.UseGlobalValues>
-  <FarmingHysteresis.UseGlobalValuesDesc>この設定が有効になっている同じ作物に指定されているすべての栽培ゾーンで,同じ境界値(下限値/上限値)が共有されます.</FarmingHysteresis.UseGlobalValuesDesc>
-
-  <FarmingHysteresis.DecrementLowerHysteresis>下限：-{0}</FarmingHysteresis.DecrementLowerHysteresis>
-  <FarmingHysteresis.DecrementLowerHysteresisDesc>下限値を{0}下げます.\n{1}キーを押すと+10,\n{2}キーを押すと+100に変更,\n{1}と{2}を同時に押すと+1000に変更</FarmingHysteresis.DecrementLowerHysteresisDesc>
-  <FarmingHysteresis.IncrementLowerHysteresis>下限：+{0}</FarmingHysteresis.IncrementLowerHysteresis>
-  <FarmingHysteresis.IncrementLowerHysteresisDesc>下限値を{0}上げます.\n{1}キーを押すと+10,\n{2}キーを押すと+100に変更,\n{1}と{2}を同時に押すと+1000に変更</FarmingHysteresis.IncrementLowerHysteresisDesc>
-
-  <FarmingHysteresis.DecrementUpperHysteresis>上限：-{0}</FarmingHysteresis.DecrementUpperHysteresis>
-  <FarmingHysteresis.DecrementUpperHysteresisDesc>上限値を{0}下げます.\n{1}キーを押すと+10,\n{2}キーを押すと+100に変更,\n{1}と{2}を同時に押すと+1000に変更</FarmingHysteresis.DecrementUpperHysteresisDesc>
-  <FarmingHysteresis.IncrementUpperHysteresis>上限：+{0}</FarmingHysteresis.IncrementUpperHysteresis>
-  <FarmingHysteresis.IncrementUpperHysteresisDesc>上限値を{0}上げます.\n{1}キーを押すと+10,\n{2}キーを押すと+100に変更,\n{1}と{2}を同時に押すと+1000に変更</FarmingHysteresis.IncrementUpperHysteresisDesc>
-
-  <FarmingHysteresis.UseGlobalBounds>{0}に全域在庫対応を使用：{1}</FarmingHysteresis.UseGlobalBounds>
-  <FarmingHysteresis.LowerBound>下限:{2}が{1}未満の場合,\n　　 {0}の種蒔きを開始</FarmingHysteresis.LowerBound>
-  <FarmingHysteresis.UpperBound>上限:{2}が{1}以上の場合,\n　　 {0}の種蒔きを停止</FarmingHysteresis.UpperBound>
-  <FarmingHysteresis.InStorage>{0}の在庫量：現在 {1}</FarmingHysteresis.InStorage>
-
-  <FarmingHysteresis.LatchModeDesc>現状:{0}</FarmingHysteresis.LatchModeDesc>
-  <FarmingHysteresis.LatchModeDesc.Unknown>不明.これはバグです.</FarmingHysteresis.LatchModeDesc.Unknown>
-  <FarmingHysteresis.LatchModeDesc.BelowLowerBound>在庫が下限未満.種蒔き可能</FarmingHysteresis.LatchModeDesc.BelowLowerBound>
-  <FarmingHysteresis.LatchModeDesc.BetweenBoundsEnabled>在庫が下限以上になり,上限未満.種蒔き可能</FarmingHysteresis.LatchModeDesc.BetweenBoundsEnabled>
-  <FarmingHysteresis.LatchModeDesc.BetweenBoundsDisabled>在庫が上限以上になった為,現在の在庫が\n　　 下限未満になるまで消費されるまで,種蒔き不可</FarmingHysteresis.LatchModeDesc.BetweenBoundsDisabled>
-  <FarmingHysteresis.LatchModeDesc.AboveUpperBound>在庫が上限以上になり.種蒔き不可</FarmingHysteresis.LatchModeDesc.AboveUpperBound>
-  <FarmingHysteresis.DisabledDueToMissingHarvestedThingDef>選択した植物に収穫可能な作物が存在しないので,この栽培ゾーンでは在庫対応が無効になっています</FarmingHysteresis.DisabledDueToMissingHarvestedThingDef>
-
-</LanguageData>
\ No newline at end of file
+﻿<?xml version="1.0" encoding="utf-8"?>
+
+<LanguageData>
+  <FarmingHysteresis.EnableFarmingHysteresis>Enable farming hysteresis</FarmingHysteresis.EnableFarmingHysteresis>

```

Comment:
Hey! Thanks for the update! Unfortunately, it looks like you've accidentally submitted the English translations in place of the Japanese ones. This whole file is just English text now.